Maddori.Apple/Maddori.Apple/Screen/MyReflection/MyReflection/MyReflectionViewController.swift

```diff
@@ -82,9 +85,10 @@ final class MyReflectionViewController: BaseViewController {
                    headers: type.headers
         ).responseDecodable(of: BaseModel<AllReflectionResponse>.self) { json in
             if let json = json.value {
-                dump(json)
+                guard let jsonDetail = json.detail else { return }
+                self.allReflection = jsonDetail
+                self.totalReflection = (self.allReflection?.reflection[0]!)!
```

Comment:
```suggestion
                guard let jsonDetail = json.detail else { return }
                if let allReflection = jsonDetail.reflection[0] {
                    self.totalReflection = allReflection
                }
```
요런식으로 가면 어떨까요..???

그리고 호야의 머리는 장식이 아닙니다.......

Reply:
따봉 !

Reply:
[ac2af8a](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/150/commits/ac2af8a6e0631034f44a461b5d68f89430d3d135) 

크... 우리 이드는 천재에용

Reply:
우리 천재는 이드에용